### PR TITLE
PD-11 Create Time Sync Article

### PR DESCRIPTION
**Pull request by micjohnson777**
This commit updates the main Dashboard article SCALEDashboard.md to include information on the System Information widget with the time sync function, and creates a new tutorial on time sync SCALETimeSync.md and navigation in tutorials for the main dashboard.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALE/SCALETutorials/Dashboard/SCALETimeSync.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+weight: 10
+aliases:
+tags:
+- scaletimesync
+---
+
+TrueNAS SCALE provides the ability to synchronize SCALE and system server time when they get out of sync with each other. 
+This function is not designed to correct time differences weeks out of alignment.
+
+The **System Information** widget on the main **[Dashboard]({{< relref "/SCALE/SCALEUIReference/SCALEDashboard.md" >}})** displays a message and provides an icon button that executes the time-synchronization operation only when SCALE detects a diescrepency between SCALE and system server time.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/Dashboard/SCALETimeSync.md:13]:
```suggestion
The **System Information** widget on the main **[Dashboard]({{< relref "/SCALE/SCALEUIReference/SCALEDashboard.md" >}})** displays a message and provides an icon button that executes the time-synchronization operation only when SCALE detects a discrepancy between SCALE and system server time.
```

File: content/SCALE/SCALEUIReference/SCALEDashboard.md [hunk 2]
```diff
@@ -1,12 +1,9 @@
 ---
 title: Main Dashboard
-description: "This article provides information on the information cards (widgets) on the **Dashboard** screen and how to customize the display by moving, adding or removing the widgets."
+description: "This article provides information on the main Dashboard information cards (widgets) and includes basic instructions on synchronizing  system server and SCALE time, and customizing the display by moving, adding, or removing the widgets."
```
Comment (tonyriv3) [inline content/SCALE/SCALEUIReference/SCALEDashboard.md:3]:
```suggestion
description: "This article provides information on the main Dashboard information cards (widgets) and includes basic instructions on synchronizing system server and SCALE time, and customizing the display by moving, adding, or removing the widgets."
```

File: content/SCALE/SCALEUIReference/SCALEDashboard.md [hunk 3]
```diff
@@ -48,7 +45,7 @@ Use **Save** to retain any setting changes you make. Click on the **X** or on an
 Click on the <span class="iconify" data-icon="ic:baseline-assessment"></span> icon to display the report screen that corresponds to that widget. For example, clicking the <span class="material-icons-outlined">assessment</span> icon on the **CPU** widget opens the **Reports > CPU** screen. 
 
 ### System Information Widget
-The **System Information** widget displays general information on the SCALE system.   
+The **System Information** widget displays general information on the SCALE system, and includes a option to syncronize the system server time with TrueNAS SCALE time if they get out of sync.
```
Comment (tonyriv3) [inline content/SCALE/SCALEUIReference/SCALEDashboard.md:48]:
```suggestion
The **System Information** widget displays general information on the SCALE system. It includes an option to synchronize the system server time with TrueNAS SCALE time if they get out of sync.   
```

**Review by tonyriv3 — CHANGES_REQUESTED** [review]
This looks great. Just a few suggested changes.

File: content/SCALE/SCALEUIReference/SCALEDashboard.md (hunk 3 above)
Comment (micjohnson777) [reply content/SCALE/SCALEUIReference/SCALEDashboard.md:48]: You know, I spelled it that way and darned if Hemingway didn't tell me I'd misspelled it!

**Review by tonyriv3 — APPROVED** [review]

**Pull request merged**